desktop_version/src/Music.cpp

```diff
@@ -1130,6 +1154,21 @@ void musicclass::playef(int t)
     soundTracks[t].Play();
 }
 
+void musicclass::playef_name(std::string& t)
+{
+    for (int i = 0; i < soundTracks.size(); i++)
+    {
+        size_t lastindex = soundTracks[i].name.find_last_of('.');
+        std::string rawname = soundTracks[i].name.substr(0, lastindex);
+        if (t == rawname)
+        {
+            soundTracks[i].Play();
+            return;
+        }
+    }
+    return;
```

Comment:
This return doesn't technically hurt but it's redundant, the function ends here either way.